### PR TITLE
Add the license notice for dependencies

### PR DESCRIPTION
**Pull request by git-hulk**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -199,3 +199,167 @@
...
+Apache Kvrocks(incubating) Dependencies:
+
+Apache Kvrocks(incubating) includes a number of Dependencies with separate copyright notices
+and license terms. Your use of these submodules is subject to the terms and
+conditions of the following licenses.
+
+For the library rocksdb(https://github.com/facebook/rocksdb):
+
+RocksDB is developed and maintained by Facebook Database Engineering Team.
+It's also under a "Apache License 2.0" license, see: https://github.com/facebook/rocksdb/blob/main/LICENSE.Apache
```
Comment (PragmaTwice) [inline LICENSE]:
```suggestion
It's also under the "Apache License 2.0" license, see: https://github.com/facebook/rocksdb/blob/main/LICENSE.Apache
```

File: LICENSE [hunk 2]
```diff
@@ -199,3 +199,167 @@
...
+    A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+    OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+    SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+    LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+    DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+    THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+    (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+    OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+For the library  googletest(https://github.com/google/googletest):
```
Comment (PragmaTwice) [inline LICENSE]:
```suggestion
For the library googletest(https://github.com/google/googletest):
```

**Review by PragmaTwice — COMMENTED** [review]
some little typos, and seems luaJIT is missing : )

**Comment by git-hulk** [comment]
> some little typos, and seems luaJIT is missing : )

Thanks twice, we won’t release luajit in 2.1.0, so will add it later.

**Review by PragmaTwice — DISMISSED** [review]

**Comment by ShooterIT** [comment]
I think we should also describe kvrocks uses some redis codes, and already put copyright on every file.

**Comment by git-hulk** [comment]
@ShooterIT Can you append the description?

**Review by PragmaTwice — DISMISSED** [review]

**Review by PragmaTwice — DISMISSED** [review]

**Review by ShooterIT — APPROVED** [review]

**Review by PragmaTwice — APPROVED** [review]

**Review by tisonkun — APPROVED** [review]

**Comment by git-hulk** [comment]
Thanks all, merging...

**Pull request merged**